src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -366,15 +370,21 @@ public StudyService(
     }
 
     Mono<Void> runLoadFlow(String studyName, String userId) {
-        Mono<UUID> networkUuid = getStudyUuid(studyName, userId);
+        Mono<Study> studyCas = getStudy(studyName, userId);
 
-        return networkUuid.flatMap(uuid -> {
+        return studyCas.flatMap(study -> {
             String path = UriComponentsBuilder.fromPath(DELIMITER + LOADFLOW_API_VERSION + "/networks/{networkUuid}/run")
-                    .buildAndExpand(uuid)
-                    .toUriString();
+                .buildAndExpand(study.getNetworkUuid())
+                .toUriString();
+
+            OutputStream body = new ByteArrayOutputStream();
+            if (study.getLoadFlowParameters() != null) {
+                JsonLoadFlowParameters.write(getLoadFlowParameters(study.getLoadFlowParameters()), body);
```

Comment:
You don't have to serialize it yourself. Just configure Webflux to add powsybl custom jackson modules and let the Spring serialize the object.
There is an example of WebFluxConfig to customize jackson modules in my PR in security-analysis-server.

Reply:
done